integration/raft/channel_participation_test.go

```diff
@@ -261,9 +264,27 @@ var _ = Describe("ChannelParticipation", func() {
 
 			By("removing orderer1 from a channel")
 			channelparticipation.Remove(network, orderer1, "participation-trophy")
+			Eventually(func() channelparticipation.ChannelList {
+				return channelparticipation.List(network, orderer1)
+			}, network.EventuallyTimeout).Should(Equal(channelparticipation.ChannelList{
+				SystemChannel: nil,
+				Channels: []channelparticipation.ChannelInfoShort{
+					{
+						Name: "another-participation-trophy",
+						URL:  "/participation/v1/channels/another-participation-trophy",
+					},
+				},
+			}))
+
+			By("submitting transaction to orderer1")
+			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello"))
+			resp, err := ordererclient.Broadcast(network, orderer1, env)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.Status).To(Equal(common.Status_BAD_REQUEST))
```

Comment:
This looks good. 

I would add a test case for removing a channel and joining it again. But let's do this in separate PR, rather then preventing this one from merging. I created this one: https://jira.hyperledger.org/browse/FAB-18319